Services/UI/OJS.Services.Ui.Data/Implementations/SubmissionsDataService.cs

```diff
@@ -31,6 +31,13 @@ public Task<IEnumerable<TServiceModel>> GetLatestSubmissions<TServiceModel>(int
             .MapCollection<TServiceModel>()
             .ToEnumerableAsync();
 
+    public async Task<int> GetTotalSubmissionsCount()
+        => await this.GetQuery(
+                orderBy: s => s.Id,
+                descending: true)
+            .Select(s => s.Id)
+            .FirstAsync();
```

Comment:
What if there are no submissions in the database? The method should return 0, not throw an exception.